packages/optimizers/css/src/CSSOptimizer.js

```diff
@@ -0,0 +1,110 @@
+// @flow strict-local
+
+import SourceMap from '@parcel/source-map';
+import {Optimizer} from '@parcel/plugin';
+// $FlowFixMe
+import {transform} from '@parcel/css';
+import {blobToBuffer} from '@parcel/utils';
+import browserslist from 'browserslist';
+
+export default (new Optimizer({
+  async optimize({
+    bundle,
+    contents: prevContents,
+    getSourceMapReference,
+    map: prevMap,
+    options,
+  }) {
+    if (!bundle.env.shouldOptimize) {
+      return {contents: prevContents, map: prevMap};
+    }
+
+    let targets = getTargets(bundle.env.engines.browsers);
+    let code = await blobToBuffer(prevContents);
+    let result = transform({
+      filename: bundle.name,
+      code,
+      minify: true,
+      source_map: !!bundle.env.sourceMap,
+      targets,
+    });
+
+    let map;
+    if (result.map != null) {
+      map = new SourceMap(options.projectRoot);
+      map.addVLQMap(JSON.parse(result.map));
```

Comment:
Ideally we'd return a parcel source map directly from @parcel/css. I think we can only return a buffer though, not an actual node source map object (they could be using different versions). But still should be faster than returning and re-parsing VLQ/JSON.

Reply:
Does the buffer have a stable format/"ABI"?